### DISCUSSION TITLE
how to specific output_dir in lazy config?

### DISCUSSION THREAD

**Discussion by lucasjinreal** [facebookresearch/detectron2#3440] [Q&A]
how to specific output_dir in lazy config?

**Comment by AKMourato** [comment]
```
cfg.train.output_dir = '...'
```

**Comment by ppwwyyxx** [comment]
You can use ` cfg.anything` as long as you read it in your training script.
The provided example script `lazyconfig_train_net.py` uses `cfg.train.output_dir` which is defined in `configs/common/train.py`.
